exception_al/exception_al/urls.py

```diff
@@ -15,8 +15,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include ('user.urls')),
+    path('api-auth/', include('rest_framework.urls')),
```

Comment:
Please add this line of code at the end for custom auth token view. (It will look different to Thinkific one)
The reason for this is without custom auth token view, when user logs in, only token is returned and it's not possible to retrieve user details only with token. We want to get token AND User object as return value.

path('api-token-auth/', CustomAuthToken.as_view()),